Mathlib/Algebra/Homology/DerivedCategory/Basic.lean

```diff
@@ -258,20 +263,13 @@ noncomputable def singleFunctorsPostcompQIso :
 lemma singleFunctorsPostcompQIso_hom_hom (n : ℤ) :
     (singleFunctorsPostcompQIso C).hom.hom n = 𝟙 _ := by
   ext X
-  dsimp [singleFunctorsPostcompQIso, HomotopyCategory.singleFunctorsPostcompQuotientIso,
-    quotientCompQhIso, HomologicalComplexUpToQuasiIso.quotientCompQhIso]
-  rw [CategoryTheory.Functor.map_id, SingleFunctors.id_hom, NatTrans.id_app]
-  erw [Category.id_comp, Category.id_comp]
+  simp [singleFunctorsPostcompQIso, HomotopyCategory.singleFunctorsPostcompQuotientIso]
   rfl
```

Comment:
Did you check why these `rfl`s are still needed?

Reply:
I'm going to leave this for someone else later.